ACadSharp/Entities/MText.Format.cs

```diff
@@ -0,0 +1,302 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace ACadSharp.Entities
+{
+	public partial class MText
+	{
+		/// <summary>
+		/// Format used by MText Tokens.
+		/// </summary>
+		public class Format
+		{
+			private float? _height;
+			private float? _width;
+			private float? _obliquing;
+			private float? _tracking;
+
+			/// <summary>
+			/// Text alignments
+			/// </summary>
+			public enum Alignment
+			{
+				/// <summary>
+				/// Align to bottom
+				/// </summary>
+				Bottom = 0,
+
+				/// <summary>
+				/// Align to center.
+				/// </summary>
+				Center = 1,
+
+				/// <summary>
+				/// Align to top.
+				/// </summary>
+				Top = 2
+			}
+
+			/// <summary>
+			/// Underline the token.
+			/// </summary>
+			public bool IsUnderline { get; set; }
+
+			/// <summary>
+			/// Overline the token.
+			/// </summary>
+			public bool IsOverline { get; set; }
+
+			/// <summary>
+			/// Strike-through the token.
+			/// </summary>
+			public bool IsStrikeThrough { get; set; }
+
+			/// <summary>
+			/// Font to use for the token.
+			/// </summary>
+			public Font Font { get; } = new Font();
+
+			/// <summary>
+			/// Text height of the token.
+			/// </summary>
+			public float? Height
+			{
+				get => this._height;
+				set => this._height = value;
+			}
+
+			/// <summary>
+			/// True if the passed height is a relative height.
+			/// </summary>
+			public bool IsHeightRelative { get; set; }
+
+			/// <summary>
+			/// Text width.
+			/// </summary>
+			public float? Width
+			{
+				get => this._width;
+				set => this._width = value;
+			}
+
+			/// <summary>
+			/// Text Obliquing
+			/// </summary>
+			public float? Obliquing
+			{
+				get => this._obliquing;
+				set => this._obliquing = value;
+			}
+
+			/// <summary>
+			/// Text Tracking
+			/// </summary>
+			public float? Tracking
+			{
+				get => this._tracking;
+				set => this._tracking = value;
+			}
+
+			/// <summary>
+			/// Vertical text alignment.
+			/// </summary>
+			public Alignment? Align { get; set; }
+
+			/// <summary>
+			/// Color of the token.
+			/// </summary>
+			public Color? Color { get; set; }
+
+			/// <summary>
+			/// Paragraph formatting values.
+			/// </summary>
+			public List<ReadOnlyMemory<char>> Paragraph { get; } = new List<ReadOnlyMemory<char>>();
+
+			/// <summary>
+			/// Creates a blank format.
+			/// </summary>
+			public Format()
+			{
+			}
+
+			/// <summary>
+			/// Creates a format with the specified simple formats.  Testing use onle.
+			/// </summary>
+			/// <param name="formats">Format string to use.</param>
+			internal Format(string formats)
```

Comment:
Same case as: https://github.com/DomCR/ACadSharp/pull/94/files#r1070666498

Can it be possible to implement this function in a `Helper` class in the test project?

Reply:
Done